tests/rptest/services/redpanda.py

```diff
@@ -556,6 +570,10 @@ def __init__(self,
             elif test_context.globals.get(self.GLOBAL_CLOUD_PROVIDER,
                                           'aws') == 'gcp':
                 self.cloud_storage_api_endpoint = 'storage.googleapis.com'
+                # For GCP, we currently use S3 compat API over boto3, which does not support batch deletes
+                # This makes cleanup slow, even for small-ish buckets.
+                # Therefore, we maintain logic of skipping bucket cleaning completely, if we are using lifecycle rule.
+                self.cloud_storage_cleanup_strategy = CloudStorageCleanupStrategy.IF_NOT_USING_LIFECYCLE_RULE
```

Comment:
So on GCP the behavior will be same as before, right? So we will not go for increased parallelism there?

Reply:
That's right, maintain existing behavior for GCP.  Reason is the additional testing involved.  Plus the fact that our deletes are not even batched on GCP right now.